scripts/auto-run.py

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env python3
+
+import argparse
+import time
+import os
+import sys
+import subprocess
+
+TENZIR = "../build/release/bin/tenzir"
```

Comment:
Consider finding the binary in `$PATH` instead.

Reply:
Definitely. This is just my local dev version of the script, I'll make the necessary changes before marking it as ready for review :+1:

Reply:
![96na3g](https://github.com/user-attachments/assets/aef0e59e-731e-4861-83d8-1f4b78f93e6c)